File: files/PHP/gant.php
<?php
class Gant
{
    private $idGant;
    private $idTasca;
    private $descripcio;
    private $idUsuari;
    
    /**
     * __construct
     *
     * @param  mixed $idGant
     * @param  mixed $idTasca
     * @param  mixed $descripcio
     * @param  mixed $idUsuari
     * @return void
     */
    public function __construct($idGant, $idTasca, $descripcio, $idUsuari)
    {
        $this->idGant = $idGant;
        $this->idTasca = $idTasca;
        $this->descripcio = $descripcio;
        $this->idUsuari = $idUsuari;
    }    
    /**
     * assignarTasca
     *
     * @return void
     */
    private function assignarTasca($idTasca){
        include_once 'conecta_desconecta.php';
        conectar();
        if ($linea = mysqli_query ($query= "UPDATE ... ;")){
            printf ("Tasca assignada");
        }
        desconectar();
    }          
    /**
     * establirDurada
     *
     * @return void
     */
    private function establirDurada(){
        include_once 'conecta_desconecta.php';
        conectar();
        if ($linea = mysqli_query ($query= "UPDATE ... ;")){
            printf ("Durada establida");
        }
        desconectar();
    }    
    /**
     * eliminarTasca
     *
     * @return void
     */
    private function eliminarTasca(){
        include_once 'conecta_desconecta.php';
        conectar();
        if ($linea = mysqli_query ($query= "DELETE ... ;")){
            printf ("Tasca eliminada");
        }
        desconectar();
    }    
    /**
     * modificarDurada
     *
     * @return void
     */
    private function modificarDurada(){
        include_once 'conecta_desconecta.php';
        conectar();
        if ($linea = mysqli_query ($query= "UPDATE ... ;")){
            printf ("Durada modificada");
        }
        desconectar();
    }    
    /**
     * desactivarTasca
     *
     * @return void
     */
    private function desactivarGant()
    {
    }
}

File: files/PYTHON/gant.py
class Gant:
    def __init__(self, idGant, idTasca, descripcio, idUsuari):
        """ constructor de clase

        Args:
            idGant (int): id de cada Gant
            idTasca (int): id de la tarea en concreto
            descripcio (string): descripción del gant
            idUsuari (int): id de usuario
        """
        self.__idGant = idGant
        self.__idTasca = idTasca
        self.__descripcio = descripcio
        self.__idUsuari = idUsuari

    def assignarTasca(self):
        """ futura función que asignará tareas
        """
        pass

    def assignarUsuari(self):
        """ futura funcion que asignará usuarios
        """
        pass

    def crearTasca(self):
        """ futura función que creará tareas
        """
        pass

    def establirDurada(self):
        """ futura función que establecerá la duración de la tarea
        """
        pass

    def eliminarTasca(self):
        """ futura función para eliminar tareas
        """
        pass

    def modificarDurada(self):
        """ futura función para modificar la duración de una tarea
        """
        pass

    def desactivarGant(self):
        """ futura función para desactivar gant
        """
        pass
